src/components/Cards/ProvidersCard.tsx

```diff
@@ -80,7 +70,10 @@ export function ProvidersCard() {
           return {
             name: o.name,
             id: o.id,
-            checked: true,
+            checked:
```

Comment:
I figured that since we never have an empty provider array in the store, (it looks like the button is disabled to save an empty store), having an empty url select everything as checked is ok. Let me know if there's something I am not thinking of here.

Reply:
Not sure I understand what you mean here, or what the implications of setting this to undefined ( i think thats whats going on?
)